api/src/wfl/module/covid.clj

```diff
@@ -481,12 +516,13 @@
   Return EXECUTOR."
   [source executor]
   (if-let [snapshot (peek-queue! source)]
-    (let [reference  (from-source executor snapshot)
-          submission (create-submission! executor reference)]
-      (write-workflows! executor reference submission)
-      (pop-queue! source)))
-  (update-terra-workflow-statuses! executor)
-  executor)
+    (let [reference        (from-source executor snapshot)
```

Comment:
Had to reorg a bit to return the executor with incremented MC version in this case, or the original executor if we're only updating active-or-failed workflow statuses.  LMK if there's a better way to do this or if this is unclear.